charts/test-v2/azuredisk-csi-driver/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
What's the reason for adding a "test-v2" chart? Generally, this should be a version number conforming to semantic versioning.

Reply:
Hmm I'm thinking, since these charts will only be deployed for testing, it makes more sense to keep these charts under the test folder and give the version number as v2.0.0-alpha.1 so that it follows semantic versioning.

Reply:
Why do we need separate charts for testing?

Reply:
With regular charts, we are deploying node and controller parts as different containers. Also we deploy the sidecar containers along with the driver. We don't needhave those for sanity and integration tests.

Also, when I tried deploying node and controller parts as separate containers, using the same tcp endpoint for both of them was giving binding problems when running on minikube. SO I created a separate helm chart which deploys a single azuredisk controller acting as both node and controller plugin.

Reply:
Right. Yes, it makes more sense to relocate these under the test directory than to keep them in the main chart repository.